corehq/apps/locations/fixtures.py

```diff
@@ -113,7 +113,7 @@ def __call__(self, restore_state):
         if not should_sync_locations(restore_state.last_sync_log, locations_queryset, restore_state):
             return []
 
-        data_fields = _get_location_data_fields(restore_user.domain)
+        data_fields = get_location_data_fields(restore_user.domain)
```

Comment:
This actually feels like part of serialization to me - since it's only used inside `get_xml_nodes`, it'd make more sense in my opinion for the call to happen there.  Then you wouldn't need to do the call in the `case_migration view` either.

Reply:
https://github.com/dimagi/commcare-hq/pull/29482/commits/486c532d759833b25edbdb732a5a9ef36515fd35